### PR TITLE
Zirkus/reverse bugfix smartgarph validators

### PR DESCRIPTION
**Pull request by mchacki**
This is a pure Jenkins circus revert PR.

It is impossible to test a branch under upgrade.
If this suite needs to server for upgrade tests we need to be able to do this.
Now we have merged something into devel which seems to not pass all of the upgrade tests.
We had to temporarily disable large portions of this framework, namely smart graphs and need to revisit a PR merged over a week ago.

This process is absolutely unsatisfying and time wasting.

### CODE REVIEW

**Comment by mchacki** [comment]
We cannot merge this PR until we have fixed devel procedure.
But we cannot test the devel procedure before this PR and the Devel PR is merged.

**Review by dothebart — APPROVED** [review]

**Pull request merged**
